charts/rabbitmq-cluster-operator/templates/messaging-topology-operator/validating-webhook-configuration.yaml

```
@@ -0,0 +1,358 @@
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and (eq .Values.msgTopologyOperator.enabled true) (not .Values.msgTopologyOperator.skipCreateAdmissionWebhookConfig) }}
+{{- $versionLabel := dict "app.kubernetes.io/version" ( include "common.images.version" ( dict "imageRoot" .Values.msgTopologyOperator.image "chart" .Chart ) ) }}
+{{- $labels := include "common.tplvalues.merge" ( dict "values" ( list .Values.commonLabels $versionLabel ) "context" . ) }}
+{{/*
+    If the user does not have cert-manager and is not providing a secret with the certificates, the chart needs to generate the secret
+  */}}
+{{- $secretName := printf "%s" (include "rmqco.msgTopologyOperator.webhook.fullname" .) }}
+{{- $ca := genCA "rmq-msg-topology-ca" 365 }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)